guacamole-ext/src/main/java/org/apache/guacamole/net/auth/DelegatingUser.java

```diff
@@ -93,11 +93,18 @@ public Date getLastActive() {
         return user.getLastActive();
     }
 
+    @Deprecated
     @Override
     public List<? extends ActivityRecord> getHistory()
```

Comment:
I've seen this applied a few times to implementations of the now-deprecated functions. Is the `@Deprecated` annotation not inherited from the interface that deprecated the overridden function?

Reply:
Yeah - if I try to compile without the `@Deprecated` annotation on anything that implements a deprecated method, I get a compiler warning, which, in my case, translates to an error, because `error: warnings found and -Werror specified`.